mantidproject/mantid
# Indirect Absorption - Update Interface

The Indirect Absorptions interface has been refactored to improve the ordering of input parameters. Default beam dimensions are now obtained in the interface rather than the algorithm. 

**To test:**
Interfaces>Indirect>Corrections>Absorption
Interface should be ordered in a similar way to that shown in the issue and units should be appropriate.
Input Files: [Corrections.zip](https://github.com/mantidproject/mantid/files/770704/Corrections.zip)


IRIS test:
Sample: `26176`
Container:`26174`
Beam should be picked up as: Width: `2.1cm` Height: `3.2cm`
Test with `Flat Plate` and use `4cm` for all fields apart from angle which should be `0`
Mass Density `1` Chemical Formula `H2-O`

ILL test:
Sample: `sample_red`
No need for container.
Beam should be: Width `3.5cm` Height `3.0cm`
Test with `Flat Plate` and use `4cm` for all fields apart from angle which should be `0`
Mass Density `1` Chemical Formula `H2-O`

Beam dimensions can also be checked in the IRIS/IN16B instrument parameter files.

Fixes #18577



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
